ckan/lib/plugins.py

```diff
@@ -91,6 +96,7 @@ def register_package_plugins(map):
     # running unit tests.
     if _default_package_plugin is not None:
         return
+    dataset = import_module(app.blueprints['dataset'].import_name)
```

Comment:
is this necessary because `from views import dataset as dataset` could do the job ?

Reply:
Nope. On that point we'll get import error for direct import

Reply:
yes, i had same issue with the import, it looks like it doesnt need the `from ckan.views...`, but  just 
`from views ...`  :)

Reply:
Yeah, but as i know, py2 relative imports are prohibited in ckan, because they work completely different in python3

Reply:
The `importlib` call and the reference used in the Blueprint creation are super cryptic.  Can't we not use a py2/py3 safe way of importing the module? I have not tested but does this help?

```
from ckan.views import dataset
```

Or maybe `from __future__ import absolute_import`?

Reply:
No, it won't work - the best I can choose is `dataset = import_module('ckan.views.dataset')`

Reply:
Currently i'm moving plugin registering/cleaning from initialization of flask stack back to routes and direct import `from .. import ..` works again.